samples/device_defender/basic_report/CMakeLists.txt

```diff
@@ -24,6 +24,14 @@ if (UNIX AND NOT APPLE)
     find_package(IotDeviceCommon-cpp REQUIRED)
     find_package(IotDeviceDefender-cpp REQUIRED)
 
-    target_link_libraries(${PROJECT_NAME} AWS::aws-crt-cpp AWS::IotDeviceCommon-cpp AWS::IotDeviceDefender-cpp)
+    if (ENABLE_SANITIZERS AND UNIX AND NOT APPLE)
```

Comment:
Would it be better to turn this into a small sharable function?

Reply:
It seems, https://github.com/aws/aws-iot-device-sdk-cpp-v2/pull/769 makes this block unnecessary.
I need to figure out how to merge these two PRs, as the number of conflicts is ridiculous.